programs/mango-v3-reimbursement/src/state/reimbursement_account.rs

```diff
@@ -0,0 +1,31 @@
+use std::mem::size_of;
+
+use anchor_lang::prelude::*;
+use static_assertions::const_assert_eq;
+
+#[account(zero_copy)]
+pub struct ReimbursementAccount {
+    pub reimbursed: u16,
+    pub claim_transferred: u16,
+    pub padding: [u8; 4],
+}
+const_assert_eq!(size_of::<ReimbursementAccount>(), 8);
+const_assert_eq!(size_of::<ReimbursementAccount>() % 8, 0);
+
+impl ReimbursementAccount {
+    pub fn reimbursed(&self, token_index: usize) -> bool {
+        self.reimbursed & (1 << token_index) == 1
+    }
+
+    pub fn mark_reimbursed(&mut self, token_index: usize) {
+        self.reimbursed |= 1 << token_index
+    }
+
+    pub fn claim_transferred(&self, token_index: usize) -> bool {
+        self.reimbursed & (1 << token_index) == 1
```

Comment:
must be `claim_transfered`

Reply:
You mean the double rr vs single r? See https://forum.wordreference.com/threads/transfered-or-transferred.2920899/#post-14766851

Reply:
haha, no, I mean there's a copy&paste error in the code, it's referencing the wrong field